pages/account.js

```diff
@@ -80,8 +80,8 @@ function Plan({ selectBilling }) {
           </tr>
           <tr>
             <td>Embed saved snippets</td>
-            <td></td>
             <td>✔</td>
+            <td></td>
```

Comment:
It needs to be checked in both the free and paid versions:

<img width="690" alt="image" src="https://user-images.githubusercontent.com/8397708/154863481-324a5579-f6ab-4581-a61c-76b925091b2c.png">

Reply:
Doh. :facepalm: